package.json

```diff
@@ -80,6 +80,7 @@
     "@types/tape": "4.13.2",
     "@types/tape-promise": "4.0.1",
     "@types/uuid": "8.3.1",
+    "@types/ws": "^7.4.7",
```

Comment:
@brioux Please lock all dependencies  down to an exact version

Also move all the dependencies to the respective package that uses it, here's a link on yarn workspaces: https://classic.yarnpkg.com/en/docs/workspaces/

Reply:
@petermetz done